asset-transfer-basic/chaincode-typescript/src/asset.ts

```diff
@@ -1,12 +1,26 @@
 /*
- * SPDX-License-Identifier: Apache-2.0
- */
+  SPDX-License-Identifier: Apache-2.0
+*/
 
+import {Object, Property} from 'fabric-contract-api';
+
+@Object()
 export class Asset {
+    @Property()
     public docType?: string;
```

Comment:
if we want to keep compatiblity with that of go-chaincode :  remove docType

Reply:
This is an optional property, Typescript is more strongly typed than Node, so it is present, but not necessary.